libs/ui-components/src/components/modals/ApproveDeviceModal/DeviceLabelMatch.tsx

```
@@ -1,68 +1,79 @@
-      return (
-        <TextInput
-          value={t('No fleet is matching the selected labels.')}
-          validated="warning"
-          readOnlyVariant="default"
-        />
-      );
+      return <StatusDisplayContent level="warning" label={t('No fleets currently match the selected labels.')} />;
     case 'checked--multiple':
+      const msg = isApproval
+        ? t('Device labels match multiple fleets, therefore the device will not be bound to any fleet.')
+        : t('Device labels match multiple fleets, therefore the device ownership will remain unchanged.');
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)